.circleci/config.yml

```diff
@@ -0,0 +1,70 @@
+version: 2
+jobs:
```

Comment:
Obviously out of scope for this PR, but I wanted to capture somewhere that we should probably use `workflows` in the future to make some of these jobs (not steps) run at the same time